contrib/storage-hive/core/src/main/java/org/apache/drill/exec/planner/sql/HivePartitionDescriptor.java

```diff
@@ -62,7 +62,7 @@ public HivePartitionDescriptor(@SuppressWarnings("unused") final PlannerSettings
       partitionMap.put(wrapper.name, i);
       i++;
     }
-    MAX_NESTED_SUBDIRS = i;
+    numPartitions = i;
```

Comment:
Is this representing the number of partitions or the number of partition levels in the hierarchy ?  If the latter (which I think is correct) could you change the name to numPartitionLevels ..